run-tests.pl

```diff
@@ -321,12 +321,6 @@ sub _run_test
       Future->wait_any(
          $f_test,
 
-         $loop->delay_future( after => 2 )
```

Comment:
Any particular reason to get rid of the "slow test" notification, rather than just make the outputting of it conditional on being a TTY, like the colouring?